adoc/admin-disaster-recovery-restore-master-nodes.adoc

```diff
@@ -0,0 +1,144 @@
+= Recover Lost Master Node
+This chapter describes how to recover {productname} master nodes.
+
+== Replace Single Master Node
+1. Remove the failed master node with skuba.
++
+Replace <NODE_NAME> with failed cluster master node name.
++
+```bash
+skuba node remove <NODE_NAME>
+```
+2. Delete failed master node from `known_hosts`.
++
+Replace <NODE_IP> with failed master node IP address.
++
+```bash
+sed -i "/<NODE_IP>/d" known_hosts
+```
+3. Prepare new instance.
+4. Use `skuba` to join master node from step 3.
++
+Replace <NODE_IP> with the new master node ip address.
++
+Replace <NODE_NAME> with the new master node name.
++
+```bash
+skuba node join --role=master --user=sles --sudo --target <NODE_IP> <NODE_NAME>
```

Comment:
Would the username always `sles`? :thinking:

Reply:
No. I will fix this. Thanks.